vle : undefined reference to pthread

Some of the vle packages can be compiled on genotoul. Error is due to undefined reference to pthread.

Eg. vle.discrete-time_test :

/home/rtrepos/save/install/vledev/lib64/libvle-2.1.a(Coordinator.cpp.o): dans la fonction « std::thread::thread<void (vle::devs::SimulatorProcessParallel::*)(), vle::devs::SimulatorProcessParallel*>(void (vle::devs::SimulatorProcessParallel::*&&)(), vle::devs::SimulatorProcessParallel*&&) »:
/tools/compilers/gcc/5.3.0/include/c++/5.3.0/thread:137: référence indéfinie vers « pthread_create »

--- a/src/vle/CMakeLists.txt
+++ b/src/vle/CMakeLists.txt
@@ -110,9 +110,10 @@ else ()
     utils/details/SpawnUnix.cpp)
 endif()
 
+option(BUILD_SHARED_LIBS "Build shared library" ON)
+
 add_library(libvle ${libvle_sources})
 
-option(BUILD_SHARED_LIBS "Build shared library" ON)
 include(GenerateExportHeader)
 generate_export_header(libvle
     EXPORT_MACRO_NAME VLE_API

